python/paddle/fluid/tests/unittests/test_conv2d_int8_mkldnn_op.py

```diff
@@ -203,26 +220,54 @@ def init_group(self):
         self.groups = 3
 
 
-#--------------------test conv2d s8 in and s8 out--------------------
-
-
-def create_test_int8_class(parent):
+def create_test_int8_class(parent, input_dt, fuse_relu):
     class TestInt8Case(parent):
         def init_dtype(self):
-            self.srctype = np.int8
-            self.dsttype = np.int8
+            if input_dt == np.uint8:
+                self.srctype = np.uint8
+            else:
+                self.srctype = np.int8
+            if fuse_relu:
+                self.dsttype = np.uint8
+            else:
+                self.dsttype = np.int8
```

Comment:
line 226-233 could be simpilfied as:
```
self.srctype = np.uint8 if input_dt == np.uint8 else np.uint8
self.dsttype = np.uint8 if fuse_relu else np.int8
```

Reply:
thanks for reminder, how about refine these code in next PR: conv with residual fusion, because it's a little change, and next PR will be merged continuously.

Reply:
done.